examples/production/tsconfig.json

```diff
@@ -0,0 +1,16 @@
+{
+  "compilerOptions": {
+    "target": "esnext",
```

Comment:
I'm curious if it's possible to get this example working with an "ES6" target, instead of "ESNext"

Reply:
The reason behind ESNext target is the concatenation with Babel. There is no reason for TypeScript to target anything lower than ESNext since Babel does the next transpilation step anyway

Reply:
Just for my own education, can you explain what babel is doing here?   (In my own personal and small typescript projects, I've been able to directly use the output from the typescript compiler, and haven't needed an additional transpilation step)

Reply:
```js
        {
          test: /\.tsx?$/,
          exclude: /node_modules/,
          use: [
            {
              loader: 'babel-loader',
              ...
            },
            {
              loader: 'awesome-typescript-loader'
            }
          ]
        }
```
Webpack does the chaining from bottom to top, so it passes the output from `awesome-typescript-loader` to `babel-loader`. Babel is just more powerful in the transpilation step and can do a lot more than TSC.
You can e.g. use [browserlist](http://browserl.ist/) via `browsers: [  '>1%'  ]` to just transpile specific features, which aren't supported by these browsers, resulting in more specialized and therefore better transpiled files.
This is also true for the polyfill step done by Babel. `useBuiltIns: 'entry'` will look into the browserlist and just give you the polyfills you really need and use. Note that I haven't added `babel-polyfill` as a dependency here, which must either be done at the index.ts file or within the entry chunk definition.